conf/conf.go

```diff
@@ -263,6 +265,17 @@ func (c *Config) Validate() error { //nolint:gocyclo
 	if c.MaxForwardWriteBatchSize < 1 {
 		return errors.NewInvalidConfigurationError("MaxForwardWriteBatchSize must be > 0")
 	}
+	if c.IntraClusterTLSConfig.Enabled {
+		if c.IntraClusterTLSConfig.CertPath == "" {
+			return errors.NewInvalidConfigurationError("IntraClusterTLSConfig.CertPath must be specified if intra cluster TLS is enabled")
+		}
+		if c.IntraClusterTLSConfig.KeyPath == "" {
+			return errors.NewInvalidConfigurationError("IntraClusterTLSConfig.KeyPath must be specified if intra cluster TLS is enabled")
+		}
+		if c.IntraClusterTLSConfig.ClientAuth != "" && c.IntraClusterTLSConfig.ClientCertsPath == "" {
```

Comment:
I believe Dragonboat only supports mTLS, not non mutual TLS, so we should drop the check `c.IntraClusterTLSConfig.ClientAuth != ""` here as as `c.IntraClusterTLSConfig.ClientCertsPath` should always be provided

Reply:
Done